backend/services/file_service.py

```
@@ -0,0 +1,31 @@
+from pathlib import Path
+import os
+import secrets
+import shutil
+from fastapi import HTTPException, UploadFile
+
+def validate_file_extension(filename: str, allowed_extensions: set):
```

Ruff (0.8.2)

Missing return type annotation for public function `validate_file_extension`

(ANN201)